implementations/rust/ockam/ockam/src/forwarder.rs

```diff
@@ -0,0 +1,96 @@
+use crate::Context;
+use ockam_core::{Address, Any, LocalMessage, Result, Route, Routed, TransportMessage, Worker};
+use tracing::info;
+
+/// Alias worker to register remote workers under local names.
+///
+/// To talk with this worker, you can use the [`RemoteForwarder`](crate::RemoteForwarder)
+/// which is a compatible client for this server.
+#[non_exhaustive]
+pub struct ForwardingService {}
+
+impl ForwardingService {
+    /// Start a forwarding service. The address of the forwarding service will be
+    /// `forwarding_service`.
```

Comment:
```suggestion
    /// `"forwarding_service"`.
```

(Just to make extra-clear it's not referring to an identifier, or something)